### ISSUE TITLE
[FIWARE] REST API: Create the endpoints for Proxy Management part (API-Umbrella)

### ISSUE THREAD

**Issue by marla-singer** [apinf/platform#2776]
Now REST API doesn't have any endpoints to manipulate with Proxy
### Goal
- Create a new swagger file for describing endpoints for Proxy Managment part
- Create an endpoint to create a proxy of type API-Umbrella. Make sure a user is admin

- Create an endpoint to returns the list of available proxy (id, name, and description of proxy)
### Parameters
- [ ] Proxy type: API-Umbrella, MQTT, Any

- Create endpoints to create a proxy backend for API-Umbrella proxy.
### Parameters:
- [ ] API ID (make sure a user has permission to edit this API)
- [ ] Proxy ID (make sure the proxy exists and type is API-Umbrella)
- [ ] API port (number type)
- [ ] Proxy backend path (make sure it is unique)
- [ ] API path
- [ ] Disabled API key (boolean value)
- [ ] Query params
- [ ] Headers
- [ ] Rate limits (Unlimited requests or Custom rate limits. Custom type can contain a lot of items)
If Custom rate limits is checked:
    - [ ] Duration, ms (number type)
    - [ ] Limit by (API key or IP Addresses)
    - [ ] Number of requests (number type)
    - [ ] Show rate limit in response headers (boolean. The true value can be used only once in the entire list)

- Create an endpoint to returns all connected API. For admin - any API, for manager - managed APIs, for ordinary - doesn't have access.

**Comment by kyyberi** [comment]
My suggestion is:
- to design API with Swagger and put it in APInf Platform under APInf organisation as "design" phase API. 
- get external developer feedback about it just like we did with APInf Plaftorm Catalog API. Someone also from FIWARE. 
- After iterating the API based on the feedback, implement the API. "Development" phase in catalog. 
- Get feedback again and finalize the API. "Testing" phase in catalog
- Release as production. "Production" phase in catalog.

**Comment by preriasusi** [comment]
@matleppa Is this still valid? Is this under a work on another issue?

**Comment by matleppa** [comment]
Related PR #3483 is waiting for review.
